getting-started/kubernetes/trying-ebpf.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Try out the eBPF tech preview
+description: Try out the eBPF dataplane tech preview.
+---
+
+### Big picture
+
+Install {{site.prodname}} and enable the tech preview of the new eBPF dataplane.
+
+> **Warning!** eBPF mode is a tech preview and should not be used in production clusters. It does not support all the features of {{site.prodname}} and it is missing some security features (for example anti-spoofing protection).
```

Comment:
Suggest we also say it has had very limited testing and will contain bugs.

Reply:
Maybe also say if you hit bugs it may result in slower performance or even dropped packets?